Installer.cpp

```diff
@@ -356,9 +364,9 @@ HRESULT NppShell::Installer::Install()
     {
         // We need to unregister the old menu on Windows 11 to prevent double entries in the old menu.
         UnregisterOldContextMenu();
-        UnregisterSparsePackage();
-
-        result = RegisterSparsePackage();
+        
+        thread reRegisterThread(ReRegisterSparsePackage);
```

Comment:
What's the point to make a thread for unregister & register the package?
What bug will it fix ?

Reply:
It fixes the issue he is having with it being done on the main thread.
He seems to know a lot more than I do about it, so I trust him in that it is the right solution.
Though I would like to have an explanation from him on why it is needed - I'll ask him in #18